packages/sdk/identity/src/odis/query.ts

```diff
@@ -38,26 +38,9 @@ export type AuthSigner = WalletKeySigner | EncryptionKeySigner
 // Re-export types and aliases to maintain backwards compatibility.
 export { AuthenticationMethod, PhoneNumberPrivacyRequest, signWithRawKey }
 
-// TODO(future, SDK) deprecate this in the next major ODIS SDK release & revisit this
```

Comment:
Since the next identity sdk release will be a major release that will take place after ODIS 2.0.0 is deployed, I don't think we need to worry about these legacy req / res types. Let me know if I'm missing something :)

Reply:
I think we should be able to bump the identity sdk up to a new major version even if the other celo sdks aren't. Also, I don't think anyone is relying on the current v2.x.x major version yet (Valora isn't) so I don't think we need to worry about this too much.

Reply:
Totally agree, just think we should give the applications team a heads up on this! :)